target/linux/mediatek/patches-5.10/421-mtd-spi-nor-add-xmc-xm25qh128c.patch

```diff
@@ -0,0 +1,11 @@
+--- a/drivers/mtd/spi-nor/xmc.c
++++ b/drivers/mtd/spi-nor/xmc.c
+@@ -14,6 +14,8 @@ static const struct flash_info xmc_parts
+ 			    SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
+ 	{ "XM25QH128A", INFO(0x207018, 0, 64 * 1024, 256,
+ 			     SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
++	{ "XM25QH128C", INFO(0x204018, 0, 64 * 1024, 256,
++			     SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
```

Comment:
This change looks good. Please send this patch also to the upstream Linux kernel mailing list at linux-mtd@lists.infradead.org to get it integrated into mainline Linux kernel. Move this to the target/linux/generic/pending-5.10/ folder in OpenWrt, it is likely that more device will use this flash chip in the future.

Reply:
Done, I will send it to the upstream later.